fixtures/no-bundle-import/src/data.txt

```diff
@@ -0,0 +1 @@
+TEST DATA
```

Comment:
I know it's a test file so it's non-blocking. 
```suggestion
TEST DATA

```

Reply:
I kinda want to keep this as-is so the inline-snapshot reads better. I'm not hugely opposed to it though—why the extra newline?